care/facility/api/serializers/facility.py

```diff
@@ -104,3 +109,29 @@ class Meta:
     def create(self, validated_data):
         validated_data["created_by"] = self.context["request"].user
         return super().create(validated_data)
+
+
+class FacilityImageUploadSerializer(serializers.ModelSerializer):
+    cover_image = serializers.ImageField(required=True, write_only=True)
+
+    class Meta:
+        model = Facility
+        fields = ("cover_image",)
+
+    def save(self, **kwargs):
+        facility = self.instance
+        image = self.validated_data["cover_image"]
+        image_name = f"{facility.external_id}_{time.time()}.jpg"
```

Comment:
are you sure you want to coerce everything to jpg?

Reply:
updated for all formats